sdk/component/vulnerability-finding/finding.go

```diff
@@ -0,0 +1,11 @@
+package vulnerability_finding
+
+import ocsf "github.com/smithy-security/smithy/sdk/gen/ocsf_schema/v1"
+
+type (
+	// VulnerabilityFinding contains data associated with findings.
+	VulnerabilityFinding struct {
```

Comment:
This was necessary as we need to have extra context for each vulnerability finding, like their id, which is useful since we have one finding per row.